CMakeLists.txt

```diff
@@ -1,6 +1,9 @@
 cmake_minimum_required(VERSION 3.5)
 
-project(Irrlicht CXX)
+project(Irrlicht
+	VERSION 1.9.0
```

Comment:
One more thing: should should be populated from `${VERSION}`
(just move a `set_property` or whatever call down below that)

Reply:
I'd like to propose another solution to this. VERSION was introduced to do exactly what ${VERSION} is doing now. I'd like to remove ${VERSION} entirely and use the provided system which is supported since version 3.0, if this is ok with you.